nistats/design_matrix.py

```diff
@@ -210,8 +214,11 @@ def _convolve_regressors(paradigm, hrf_model, frame_times, fir_delays=[0],
     regressor_names = []
     regressor_matrix = None
     if hrf_model == 'fir':
+        if oversampling not in [1, None]:
+            warn('Forcing oversampling factor to 1 for a finite'
+                 'impulse response hrf model')
```

Comment:
Can you increase the stacklevel here ? So that it won't float around other verbose messages.